Docs: Troubleshooting update (#16721)
elastic/logstash: docs/static/troubleshoot/ts-logstash.asciidoc

-Djdk.io.File.enableADS=true
-----

[[ts-container-cgroup]]
===== Container exits with 'An unexpected error occurred!'

> Logstash may not start when running on a container.

Reviewer: ```suggestion
Logstash may not start when running on a container due to a bug in the JDK https://bugs.openjdk.org/browse/JDK-8343191.
```

> Logstash may not start when running on a container due to a bug in the JDK https://bugs.openjdk.org/browse/JDK-8343191. A Pull Request was created to fix the issue: https://github.com/openjdk/jdk/pull/21808.

Reviewer 2: ```suggestion
{ls} running in a container may not start due to a https://bugs.openjdk.org/browse/JDK-8343191[bug in the JDK]. 
```

Reviewer 2: We can future proof the doc by removing the link to the in process PR.  The issue you link to contains status info. 
<img width="894" alt="Screenshot 2024-11-26 at 4 08 22 PM" src="https://github.com/user-attachments/assets/baa36d4d-a3c6-456c-a2cc-3458068ad290">

Please LMKWYT.
